### DISCUSSION TITLE
Utility Functions

### DISCUSSION THREAD

**Discussion by bright2kwame** [Samsonroyal/clearpath_backend#14] [Ideas]
Someone can lead and build a utility function that only prints messages in dev but does not do so in production and make it available for everyone to use @Samsonroyal

**Comment by Samsonroyal** [comment]
> Someone can lead and build a utility function that only prints messages in dev but does not do so in production and make it available for everyone to use @Samsonroyal

Hello Bright; I am intrigued how that would look like. Let's discuss this more tomorrow.
I believe we can flesh it out more.
